pycbc/workflow/core.py

```diff
@@ -411,11 +411,11 @@ def output_map(self):
         return path
         
 
-    def execute_node(self, node):
+    def execute_node(self, node, verbatim_exe = False):
         """ Execute this node immediately on the local machine
         """
         node.executed = True
-        cmd_list = node.get_command_line()
+        cmd_list = node.get_command_line(verbatim_exe)
```

Comment:
I don't like supplying kwargs without the kwarg specified. Is this just me? Alex likes to do this too. For me this should be
```
cmd_list = node.get_command_line(verbatim_exe=verbatim_exe)
```
@cmbiwer is normally our code health guru! Unless you feel strongly, can you fix this?